src/App.tsx

```diff
@@ -1,15 +1,52 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [typingTimeout, setTypingTimeout] = useState<number | undefined>(
```

Comment:
According to the check-list suggestions should appear after some delay in typing (debounce); So you need to use debounce function ( or your own function  or import debounce from  'lodash.debounce'

Reply:
According to the check-list the delay should be customizable via props (default value is 300ms);